hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/RepositoryConfigurationView.java

```diff
@@ -144,4 +158,66 @@ public void disableMultipleAssignmentOption() {
         multiAssignmentsConfigurationItem.setEnabled(false);
     }
 
+    @Override
+    public void save() {
+        writeConfigOption(TenantConfigurationProperties.TenantConfigurationKey.ACTION_CLEANUP_ENABLED,
```

Comment:
remove qualifier to improve readability

Reply:
Done. Just calling `TenantConfigurationKey.ACTION_CLEANUP_ENABLED` now